src/commands/convert/asff2hdf.ts

```diff
@@ -19,37 +23,118 @@ export default class ASFF2HDF extends Command {
+        } else {
+          logger.error('Invalid ASFF findings format')
+        }
+      } catch {
+        findings.push(...data.split('\n'))
+      }
+
+      // If we've been passed a Security Standards JSON
+      if (flags.securityhub) {
+        securityhub = flags.securityhub.map(file =>
+          fs.readFileSync(file, 'utf-8'),
+        )
+      }
+    } else if (flags.aws) { // Flag to pull findings from AWS Security Hub
+      AWS.config.update({
+        httpOptions: {
+          agent: new https.Agent({
+            // Disable HTTPS verification if requested
+            rejectUnauthorized: !flags.insecure,
+            // Pass an SSL certificate to trust
+            ca: flags.certificate ? fs.readFileSync(flags.certificate, 'utf-8') : undefined,
+          }),
+        },
+      })
+      const clientOptions: AWS.SecurityHub.ClientConfiguration = {
+        region: flags.region,
+      }
+      // Create our SecurityHub client
+      const client = new AWS.SecurityHub(clientOptions)
+      // Pagination
+      let nextToken = null
+      let filters: AwsSecurityFindingFilters = {}
+
+      // Filter by target name
+      if (flags.target) {
+        filters = {
+          Id: flags.target.map(target => {
+            return {Value: target, Comparison: 'PREFIX'}
+          }),
+        }
+      }
+
+      logger.info('Starting collection of Findings')
+      const queryParams = {Filters: filters, MaxResults: 100}
+      // Get findings
+      while (nextToken !== undefined) {
+        logger.debug(`Querying for NextToken: ${nextToken}`)
+        _.set(queryParams, 'NextToken', nextToken)
+        const getFindingsResult = await client.getFindings(queryParams).promise()
+        logger.debug(`Received: ${getFindingsResult.Findings.length} findings`)
+        findings.push(...getFindingsResult.Findings.map(finding => JSON.stringify(finding)))
+        nextToken = getFindingsResult.NextToken
+      }
+
+      nextToken = null
+
+      logger.info('Starting collection of security standards')
+      const standards: AWS.SecurityHub.GetEnabledStandardsResponse = {
+        StandardsSubscriptions: [],
+      }
+
+      // Get acive security standards subscriptions (enabled standards)
+      while (nextToken !== undefined) {
+        logger.debug(`Querying for NextToken: ${nextToken}`)
+        _.set(queryParams, 'NextToken', nextToken)
+        const getStandardsResult = await client.getEnabledStandards().promise()
+        logger.debug(`Received: ${getStandardsResult.StandardsSubscriptions?.length} standards`)
+        if (getStandardsResult.StandardsSubscriptions) {
+          standards.StandardsSubscriptions?.push(...getStandardsResult.StandardsSubscriptions)
+          nextToken = getStandardsResult.NextToken
+        } else {
+          logger.debug('No more enabled standards found')
+          break
+        }
+      }
+
+      // Store for the HDF converter
+      securityhub = [JSON.stringify(standards)]
```

Comment:
This is putting the wrong data in there.  The getEnabledStandards only returns metadata for the standard and not the standard itself.  You need to extract the StandardsSubscriptionArn from the metadata object and then call describeStandardsControls with that passed in.  Each object returned from calling that per standard should be added to the 'securityhub' array.